src/components/Banner.jsx

```diff
@@ -0,0 +1,12 @@
+import "../index.css";
+import React from "react";
+import { Link } from "react-router-dom";
+import { BannerImage } from "../assets";
+
+function Banner() {
+  return (
+    <img className="home_banner" src={BannerImage} alt="" />
```

Comment:
One Suggesstion, it would be better if you give alternate text for an image

Reply:
Done